fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -0,0 +1,80 @@
+package com.gu.facia.api
+
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.facia.api.contentapi.ContentApi
+import com.gu.facia.api.contentapi.ContentApi.{AdjustItemQuery, AdjustSearchQuery}
+import com.gu.facia.api.models._
+import com.gu.facia.client.ApiClient
+import com.gu.facia.client.models.{CollectionConfigJson, TrailMetaData}
+
+import scala.concurrent.ExecutionContext
+
+
+object FAPI {
```

Comment:
As `capiClient` and `faciaClient` are required for every method here, why is this not just a `class` where those are passed in as parameters on construction?